app/models/apple/key.rb

```diff
@@ -8,6 +8,16 @@ class Key < ApplicationRecord
 
     validate :provider_id_is_valid, if: :provider_id?
     validate :ec_key_format, if: :key_pem_b64?
+    validate :must_have_working_key
```

Comment:
@svevang here's the validation for the key - it tries getting a list of shows, and fails the validation on any kind of error i.e. fails on 401s which raise an error